### PR TITLE
Fixed(?) set up of default packeg name from jdkMajor

### PR DESCRIPTION
**Pull request by judovana**

### CODE REVIEW

File: webapp/src/components/formComponents/JDKVersionSelectForm.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const ProductSelectForm: React.FC<Props> = props => {
 
         const onJDKVersionChange = (value: string) => {
             product.jdk = value
+            product.packageName = jdkVersion!.packageNames.length >= 0 ? jdkVersion!.packageNames[0] : ""
```
Comment (judovana) [inline webapp/src/components/formComponents/JDKVersionSelectForm.tsx]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Unexpected_type

Comment (oklinov) [reply webapp/src/components/formComponents/JDKVersionSelectForm.tsx]: It should be packageNames.length > 0, not >= ... my bad, could you please fix it? :)

Comment (judovana) [reply webapp/src/components/formComponents/JDKVersionSelectForm.tsx]: Well, sure, but this patch is generally not working. Not product, not jdk verson, not anything do exists here in time of call. You may see the desperate ! in the line. What is the variable to use?

Comment (judovana) [reply webapp/src/components/formComponents/JDKVersionSelectForm.tsx]: fixed, the >; however the PR still do not work.I humbly ask for your assistance on this my first react code.

File: webapp/src/components/formComponents/JDKVersionSelectForm.tsx [hunk 2]
```diff
@@ -21,7 +21,11 @@ const ProductSelectForm: React.FC<Props> = props => {
 
         const onJDKVersionChange = (value: string) => {
             product.jdk = value
-            product.packageName = jdkVersion!.packageNames.length > 0 ? jdkVersion!.packageNames[0] : ""
+            const jdkVersion = getJDKVersion(product.jdk)
+            if (jdkVersion) {
+                const packageNames = jdkVersion.packageNames
+                product.packageName = packageNames.length > 0 ? packageNames[0] : ""
+            }
```
Comment (judovana) [inline webapp/src/components/formComponents/JDKVersionSelectForm.tsx]: I see. Thanx!

**Comment by oklinov** [comment]
Fixes issue #116

**Pull request merged**